package.json

```diff
@@ -17,9 +17,9 @@
     "test:setup": "tests/scripts/test_setup.sh",
     "test:unit": "node tests/jsunit/test_runner.js",
     "test:lint": "eslint .",
-    "test": "npm run test:lint && npm run test:setup && npm run test:unit",
+    "test": "npm run test:lint && npm run test:setup && npm run test:unit && npm run translate",
```

Comment:
Maybe the translate step should run before the unit tests (and perhaps before the setup), so that it can fail early for simple failures like using `'` instead of `"`.